File: 2206 divide-array-into-equal-pairs/solution.java
class Solution {
    public boolean divideArray(int[] nums) {
        int[] f = new int[600];
        for(int i:nums) f[i]++;
        for(int i:nums) {
            if((f[i] & 1) == 1) return false;
        }
        return true; 
    }
}

File: 2206 divide-array-into-equal-pairs/solution.cs
public class Solution {
    public bool DivideArray(int[] nums) {
        int[] f = new int[600];
        foreach(int i in nums) f[i]++;
        foreach(int i in nums) {
            if((f[i] & 1) == 1) return false;
        }
        return true; 
    }
}
